index.html

```diff
@@ -0,0 +1,496 @@
+          <p>
+            With each iteration, the loop increments n and adds that value to x.
+            Therefore, x and n take on the following values:
+          </p>
+          <ul>
+            <li>After the first pass: n = 1 and x = 1</li>
+            <li>After the second pass: n = 2 and x = 3</li>
+            <li>After the third pass: n = 3 and x = 6</li>
+          </ul>
+          <p>
+            After completing the third pass, the condition n < 3 is no longer
+            true, so the loop terminates.
+          </p>
+        </div>
+      </section>
+
+      <section class="main-section" id="FUNCTION_DECALARATIONS">
+        <header>Function Declarations</header>
+        <div>
+          <p>
+            A function definition (also called a function declaration, or
+            function statement) consists of the function keyword, followed by:
+          </p>
+          <ul>
+            <li>The name of the function.</li>
+            <li>
+              A list of arguments to the function, enclosed in parentheses and
+              separated by commas.
+            </li>
+            <li>
+              The JavaScript statements that define the function, enclosed in
+              curly brackets, { }.
+            </li>
+          </ul>
+          <p>
+            For example, the following code defines a simple function named
+            square:
+          </p>
+          <code>function square(number) { return number * number; } </code>
+          <p>
+            The function square takes one argument, called number. The function
+            consists of one statement that says to return the argument of the
+            function (that is, number) multiplied by itself. The return
+            statement specifies the value returned by the function.
+          </p>
+          <code>return number * number; </code>
+          <p>
+            Primitive parameters (such as a number) are passed to functions by
+            value; the value is passed to the function, but if the function
+            changes the value of the parameter, this change is not reflected
+            globally or in the calling function.
+          </p>
+        </div>
+      </section>
+
+      <section class="main-section" id="Reference">
+        <header>Reference</header>
+        <div>
+          <ul>
+            <li>
+              All the documentation in this page is taken from
+              <a
+                href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide"
+                target="_blank"
+              >
+                MDN</a
+              >
+            </li>
+          </ul>
+        </div>
+      </section>
+    </main>
+
+    <nav id="navbar">
+      <header>JS Documentation</header>
+      <ul>
+        <li><a class="nav-link" href="#Introduction"> Introduction</a></li>
+        <li>
+          <a class="nav-link" href="#What_you_should_already_know"
+            >WHAT YOU SHOULD ALREADY KNOW</a
+          >
```

Comment:
Kindly move this closing symbol to a tag.

Reply:
@amare1990 kindly try to include an explanation "why" for your feedback, to help the student understand why you requesting the changes